javatests/arcs/core/host/TestPlannedParticle.kt

```diff
@@ -0,0 +1,23 @@
+package arcs.core.host
+
+import arcs.core.data.Plan
+import arcs.core.data.Schema
+import arcs.core.entity.EntityBaseSpec
+import arcs.sdk.BaseParticle
+import arcs.sdk.EntityBase
+import arcs.sdk.HandleHolderBase
+import arcs.sdk.ReadCollectionHandle
+
+open class TestPlannedParticle(val spec: Plan.Particle?) : BaseParticle() {
```

Comment:
As asked earlier - can we test the this particle actually get the schema it is declared with? That's what I think we need to land this PR.

Reply:
Yes -- I added a test in this most recent set of commits.

Reply:
Maybe I misunderstand -- what do you mean by schema that it is declared with?